content/applications/finance/accounting/reporting.rst

```diff
@@ -7,9 +7,8 @@ Reporting
 Main reports available
```

Comment:
This H1 title is the only one in this page, I would remove it.

Reply:
There must always be an H1